.gitignore

```diff
@@ -19,5 +19,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
-/package.json
 /package-lock.json
```

Comment:
package lock needs to be committed into the repo. please look into why that should be